www/src/components/repository/RepositoryDescription.js

```diff
@@ -1,6 +1,35 @@
+import { useContext } from 'react'
+import { Flex, H2, P } from 'honorable'
+
+import RepositoryContext from '../../contexts/RepositoryContext'
+
+import RepositoryDescriptionMarkdown from './RepositoryDescriptionMarkdown'
+
 function RepositoryDescription() {
+  const repository = useContext(RepositoryContext)
+
   return (
-    'RepositoryDescription'
+    <Flex
+      direction="column"
+      height="100%"
+    >
+      <H2>
+        Description
+      </H2>
+      <Flex
+        mt={2}
+        direction="column"
+        flexGrow={1}
+        maxWidth="700px"
+      >
+        {repository.readme ? (
+          <RepositoryDescriptionMarkdown
+            text={repository.readme}
+            gitUrl={repository.git_url}
+          />
+        ) : <P>No description available</P>}
```

Comment:
This should be wrapped in parenthesis and put on a new line. Weird that the linter can't catch that...